src/common/mfu_util.c

```diff
@@ -195,6 +197,45 @@ int daos_connect(
 
     return 0;
 }
+
+int daos_mount(
+  mfu_file_t* mfu_file,
+  daos_handle_t* poh,
+  daos_handle_t* coh)
+{
+    int rc = 0;
+
+    /* Mount dfs */
+    rc = dfs_mount(*poh, *coh, O_RDWR, &mfu_file->dfs);
+    if (rc != 0) {
+        MFU_LOG(MFU_LOG_ERR, "Failed to mount DAOS filesystem (DFS): "
+                MFU_ERRF, MFU_ERRP(-MFU_ERR_DAOS));
```

Comment:
I haven't checked these macros, but do we need corresponding printf-type codes up in the error string, like %d and %s?

Reply:
I modeled this after the way DAOS does it, albeit simplified:
```
#define MFU_ERRF "%s(%d)"
#define MFU_ERRP(rc) "MFU_ERR", rc
```
Means that:
```
MFU_LOG(MFU_LOG_ERR, "Failed to mount DAOS filesystem (DFS): "
                MFU_ERRF, MFU_ERRP(-MFU_ERR_DAOS));
```
Evaluates to:
```
MFU_LOG(MFU_LOG_ERR, "Failed to mount DAOS filesystem (DFS): "
                "%s(%d)", "MFU_ERR", -MFU_ERR_DAOS);
```
Simplified:
```
MFU_LOG(MFU_LOG_ERR, "Failed to mount DAOS filesystem (DFS): %s(%d)"
                "MFU_ERR", -MFU_ERR_DAOS);
```

In the future, if desired, we could change:
```
#define MFU_ERRP(rc) "MFU_ERR", rc
```
To instead show the string corresponding to that rc value, instead of just always "MFU_ERR". DAOS does this, but as you can imagine, associating a string with an rc value, and the rc value with the same string, through macros, will take a little bit more work. DAOS handles this by have macros that "register" each error code with a string.

Reply:
Ah, nice.  Yes, that's all fine then.

Thanks for the explanation.